prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -156,6 +156,19 @@ class SettingsFacade(
         }
     }
 
+    /**
+     * Queries for the settings history for an org based on the type (RECEIVER, SENDER, ORG)
+     * The whole history (incl inactive and deleted) returned across all names for a given type.
+     * @param organizationName Restrict query to this org
+     * @param settingsType Type of setting. column in db and used to select
+     */
+    fun findSettingHistoryAsJson(organizationName: String, settingsType: SettingType): String {
+        val settings = db.transactReturning { txn ->
+            db.fetchSettingRevisionHistory(organizationName, settingsType, txn)
+        }
+        return if (settings.isNotEmpty()) mapper.writeValueAsString(settings) else "[]"
```

Comment:
Instead of "[]" use a Json library to write an empty array

Reply:
Seems heavyweight since it just generates "[]", but will do.

Reply:
Just doing a `return mapper.writeValueAsString(settings)` does the right thing and returns `[ ]` and is easier to read so 👍